documentation/site/content/samples/azure-kubernetes-service/model-in-image.md

```diff
@@ -696,15 +696,15 @@ Events:             <none>
 Access the Administration Console using the admin load balancer IP address.
 
 ```shell
-$ ADMIN_SERVER_IP=$(kubectl -n sample-domain1-ns get svc sample-domain1-admin-server-external-lb -o=jsonpath='{.status.loadBalancer.ingress\[0\].ip}')
+$ ADMIN_SERVER_IP=$(kubectl -n sample-domain1-ns get svc sample-domain1-admin-server-external-lb -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
Hello @edburns the command failed on Ubuntu, Zhihao help verify the updated command on MacOS, see https://dev.azure.com/weblogic-azure/Open%20Standard%20Enterprise%20Java%20(Java%20EE)%20on%20Azure/_workitems/edit/452